app/classifier/tasks/multiple.jsx

```diff
@@ -0,0 +1,202 @@
+import { Markdown } from 'markdownz';
+import React from 'react';
+import GenericTask from './generic';
+import GenericTaskEditor from './generic-editor';
+
+const NOOP = Function.prototype;
+
+class MultipleChoiceSummary extends React.Component {
+  constructor(props) {
+    super(props);
+    this.expand = this.expand.bind(this);
+    this.collapse = this.collapse.bind(this);
+    this.state = {
+      expanded: this.props.expanded
+    };
+  }
+
+  expand() {
+    this.setState({ expanded: true });
+  }
+
+  collapse() {
+    this.setState({ expanded: false });
+  }
+
+  render() {
+    let toggleButton = <button type="button" className="toggle-more" onClick={this.expand}>More</button>;
+    let answers = <div className="answer">No answer</div>;
```

Comment:
Setting `[<div className="answer">No answer</div>]` as the default value of answers might be cleaner here. It would make sure that `answers` is always an array.